### DISCUSSION TITLE
How may I setup Product description like to table view?

### DISCUSSION THREAD

**Discussion by mamunahmed-m360ict** [medusajs/medusa#5034] [Q&A]
How may I setup Product description like to table view?
![image](https://github.com/medusajs/medusa/assets/122720885/944fea8c-85ad-4412-9494-38838ecd42c8)

Supposed I have a product which has lot of feature like to above image, So at this situation how may I setup in backend or front store in Medusa js?

**Comment by tsvetann** [comment]
You have to implement it yourself or integrate CMS like strapi

**Comment by pevey** [comment]
You could add all that data in metadata.  If you have tons of products, that could affect performance, though.

**Reply by octalpixel** [reply]
Hey @pevey thinking out loud, would making it a JSON string make a difference in performance ?
